### PR TITLE
[Feature] Support logprobs_mode

### PR DESCRIPTION
**Pull request by ckl117**
## Motivation

Support specifying the content returned by logprobes by adding the startup parameter `--logprobs-mode`, default is raw_logprobs. 

Options: raw_logprobs, processed_logprobs, raw_logits, processed_logits

Raw means the values before applying logit processors, like bad words. Processed means the values after applying such processors.

TODO: Speculative decoding support processed logprob.

## Usage or Command

```
python -m fastdeploy.entrypoints.openai.api_server \
    --model "/models/ERNIE-4.5-0.3B-Paddle" \
    --tensor-parallel-size 1 \
    --port 8112 \
    --enable-logprob \
    --logprobs-mode raw_logits
```

## Checklist

- [x] Add at least a tag in the PR title.
  - Tag list: [`[FDConfig]`,`[APIServer]`,`[Engine]`, `[Scheduler]`, `[PD Disaggregation]`, `[Executor]`, `[Graph Optimization]`, `[Speculative Decoding]`, `[RL]`, `[Models]`, `[Quantization]`, `[Loader]`, `[OP]`, `[KVCache]`, `[DataProcessor]`, `[BugFix]`, `[Docs]`, `[CI]`, `[Optimization]`, `[Feature]`, `[Benchmark]`, `[Others]`, `[XPU]`, `[HPU]`, `[GCU]`, `[DCU]`, `[Iluvatar]`, `[Metax]`]
  - You can add new tags based on the PR content, but the semantics must be clear.
- [x] Format your code, run `pre-commit` before commit.
- [x] Add unit tests. Please write the reason in this PR if no unit tests.
- [x] Provide accuracy results.
- [x] If the current PR is submitting to the `release` branch, make sure the PR has been submitted to the `develop` branch, then cherry-pick it to the `release` branch with the `[Cherry-Pick]` PR tag.

### CODE REVIEW

File: fastdeploy/engine/args_utils.py [hunk 1]
```diff
@@ -612,6 +623,12 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
             default=EngineArgs.enable_logprob,
             help="Enable output of token-level log probabilities.",
         )
+        model_group.add_argument(
+            "--logprobs-mode",
+            type=str,
+            default=EngineArgs.logprobs_mode,
+            help="Indicates the content returned in the logprobs.",
+        )
```
Comment (yuanlehome) [inline fastdeploy/engine/args_utils.py:632]: 需要添加参数配置非法拦截报错

Comment (yuanlehome) [reply fastdeploy/engine/args_utils.py:632]: 另外logprobs相关的配置是对齐vllm的不

Comment (ckl117) [reply fastdeploy/engine/args_utils.py:632]: 已添加非法参数拦截。配置是对齐vllm，只是当前还不支持投机解码场景下根据后处理后的logits计算logprob。

**Review by gongshaotian — APPROVED** [review]
LGTM
